### PR TITLE
Release v7.3.0

### PR DESCRIPTION
**Pull request by lildude**
Time for a new release and this time we should have no missing languages 😁.

I've tested creating this release using the changes proposed in https://github.com/github/linguist/pull/4451.

Once deployed (hopefully tomorrow or Thursday, depends on the deploy queue), this will bring back Solidity syntax highlighting which broke in the last release due to an unexpected change in the grammar and brought to our attention at https://github.com/github/linguist/pull/3973#issuecomment-461797745 and pulls in support for new languages and the other changes that have happened since v7.2.0.

/cc @asymmetric @veox @davidhq

### CODE REVIEW

File: test/test_heuristics.rb [hunk 1]
```diff
@@ -73,7 +73,7 @@ def test_asc_by_heuristics
       "AsciiDoc" => all_fixtures("AsciiDoc"),
       "AGS Script" => all_fixtures("AGS Script"),
       "Public Key" => all_fixtures("Public Key")
-    }, alt_name="test.asc")
+    }, "test.asc")
```
Comment (lildude) [inline test/test_heuristics.rb]: This resolves these warnings:

```
/Users/lildude/github/linguist/test/test_heuristics.rb:76: warning: assigned but unused variable - alt_name
```

File: .gitmodules [hunk 2]
```diff
@@ -148,6 +148,7 @@
 [submodule "vendor/grammars/SublimeEthereum"]
 	path = vendor/grammars/SublimeEthereum
 	url = https://github.com/davidhq/SublimeEthereum.git
+	branch = linguist
```
Comment (lildude) [inline .gitmodules]: Locks us to the linguist branch as per https://github.com/github/linguist/pull/3973#issuecomment-461932742

**Comment by pchaigno** [comment]
@lildude Any chance you could add #4459 to this release?

File: .gitmodules (hunk 2 above)
Comment (Alhadis) [reply .gitmodules]: Could you pull in the just-merged changes from #4459? This was supposed to be included with #4445 but I screwed up. 😭

**Comment by lildude** [comment]
> @lildude Any chance you could add #4459 to this release?

... and ...

> Could you pull in the just-merged changes from #4459? This was supposed to be included with #4445 but I screwed up. 😭

Ooof, sadly no. The GitHub.com deploy servers already have the v7.3.0 gem waiting to roll out. I can't replace it with another one of the same version due to a pretty aggressive caching mechanism. This is going to have to wait for the next release.  I'll try get a 7.3.1 out later this week or early next to pull in https://github.com/github/linguist/pull/4459

**Comment by lildude** [comment]
Linguist v7.3.0 is now live on GitHub.com and Solidity files are being highlighted again. Merging this and will get into the queue for a v7.3.1 release to pull in https://github.com/github/linguist/pull/4459

**Pull request merged**